app/styles/play/modal/announcement-modal.sass

```diff
@@ -0,0 +1,208 @@
+@import "app/styles/mixins"
+@import "app/styles/bootstrap/variables"
+
+#announcement-modal
+  //- Clear modal defaults
+  .modal-dialog
+    margin: 60px auto 0 auto
+    padding: 0
+    width: 746px
+    height: 520px
+    background: none
+
+  .modal-content
+    box-shadow: none
+    height: 520px
+
+  //- Background
+  #subscribe-background
+    position: absolute
+    top: -61px
+    left: 0px
+
+
+  //- Header
+  h1
+    position: absolute
+    left: 170px
+    top: 25px
+    margin: 0
+    width: 410px
+    text-align: center
+    color: rgb(254,188,68)
+    font-size: 38px
+    text-shadow: black 4px 4px 0, black -4px -4px 0, black 4px -4px 0, black -4px 4px 0, black 4px 0px 0, black 0px -4px 0, black -4px 0px 0, black 0px 4px 0, black 6px 6px 6px
+    font-variant: normal
+    text-transform: uppercase
+
+
+  //- Close modal button
+  #close-modal
+    position: absolute
+    left: 568px
+    top: 17px
+    width: 60px
+    height: 60px
+    color: white
+    text-align: center
+    font-size: 30px
+    padding-top: 15px
+    cursor: pointer
+    @include rotate(-3deg)
+
+    &:hover
+      color: yellow
+
+  .subscribe-image
+    position: absolute
+    top: 114px
+    right: 65px
+    
+  //- Paper area
+  .sub
+    position: absolute
+    right: -20px
+    bottom: -32px
+    z-index: 1
+    width: 256px
+
+    button
+      width: 256px
+
+  .shimmy
+    animation: shimmy-and-shake 0.25s 2 alternate
+
+  @keyframes shimmy-and-shake
```

Comment:
Renamed it to 'excited wiggle' and the class to 'wiggle' so it more descriptive.

A shimmy is more of a slide back and forth, but a wiggle is a bit more of a in-place squirm.